packages/terra-tabs/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Added
   * Added prop to support the focus traverse within the tab content.
+  * Fix for Active Key Style issue.
```

Comment:
can we move this entry under Fixed section instead of Added
 * Fixed
   *  Fixed tab active indicator issue

Reply:
The entry is too vague. What is the issue? We should write it as:

`Fixed <description of bug>`

Reply:
Added.